apps/site/src/app/apply/sections/Components/SimpleRadio.tsx

```diff
@@ -0,0 +1,59 @@
+import RequiredAsterisk from "./RequiredAsterisk";
+
+interface TextfieldProps {
+	name: string;
+	values: Array<{
+		id: string;
+		labelText: string;
+		inputValue: string;
+	}>;
+	title: string;
+	titleClass: string;
+	containerClassTotal: string;
+	containerClassInputLabels: string;
+	containerClassValues: string;
+	labelClass: string;
+	isRequired: boolean;
+}
+
+export default function SimpleRadio({
+	name,
+	values,
+	title,
+	titleClass,
+	containerClassTotal,
+	containerClassInputLabels,
+	containerClassValues,
+	labelClass,
+	isRequired,
+}: TextfieldProps) {
+	return (
+		<div className={containerClassTotal}>
+			<p className={titleClass}>
+				{`${title} `}
+				{!isRequired || <RequiredAsterisk />}
```

Comment:
Nitpick: using negation with disjunction mangles the interpretability a little. A direct conjunction could be easier to understatnd
```suggestion
				{isRequired && <RequiredAsterisk />}
```